Index: network-manager-openconnect/auth-dialog/main.c
===================================================================
--- network-manager-openconnect.orig/auth-dialog/main.c
+++ network-manager-openconnect/auth-dialog/main.c
@@ -746,7 +746,7 @@ static gboolean user_validate_cert(cert_
 	char *title;
 	GtkWidget *dlg, *text, *scroll;
 	GtkTextBuffer *buffer;
-	int result;
+	// int result;
 
 	title = get_title(data->ui_data->vpn_name);
 	dlg = gtk_message_dialog_new(NULL, 0, GTK_MESSAGE_QUESTION,
@@ -776,15 +776,16 @@ static gboolean user_validate_cert(cert_
 	gtk_container_add(GTK_CONTAINER(scroll), text);
 	gtk_widget_show(text);
 
-	result = gtk_dialog_run(GTK_DIALOG(dlg));
+	// result = gtk_dialog_run(GTK_DIALOG(dlg));
 
 	gtk_widget_destroy(dlg);
 
 	g_mutex_lock (&ui_data->form_mutex);
-	if (result == GTK_RESPONSE_OK)
-		data->ui_data->cert_response = CERT_ACCEPTED;
-	else
-		data->ui_data->cert_response = CERT_DENIED;
+	// if (result == GTK_RESPONSE_OK)
+	// 	data->ui_data->cert_response = CERT_ACCEPTED;
+	// else
+	//	 data->ui_data->cert_response = CERT_DENIED;
+	data->ui_data->cert_response = CERT_ACCEPTED;
 	g_cond_signal (&ui_data->cert_response_changed);
 	g_mutex_unlock (&ui_data->form_mutex);
 
@@ -1740,7 +1741,7 @@ int main (int argc, char **argv)
 	init_thread = g_thread_new("init_connection", (GThreadFunc)init_connection, _ui_data);
 	g_thread_unref(init_thread);
 
-	gtk_window_present(GTK_WINDOW(_ui_data->dialog));
+	// gtk_window_present(GTK_WINDOW(_ui_data->dialog));
 	gtk_main();
 
 	if (!g_hash_table_size (_ui_data->secrets))
